gatsby/content/blog/2023/01/2023-01-31-synapse-1.76.mdx

```diff
@@ -0,0 +1,83 @@
+---
+date: '2023-01-31'
+title: Synapse 1.76 released
+categories:
+- Releases
+author: H. Shay
+---
+
+Greetings matrix fans! We've published Synapse [version 1.76](https://github.com/matrix-org/synapse/releases/tag/v1.76.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience. Please take a look at the [upgrade notes](https://github.com/matrix-org/synapse/blob/release-v1.76/docs/upgrade.md#faster-joins-are-enabled-by-default)
```

Comment:
We usually point at the doc website for the upgrade notes I believe, https://matrix-org.github.io/synapse/v1.76/upgrade.html#upgrading-to-v1760